Update RBF mapping reference

davidscn opened the pull request:
The RBF mapping description is much more exhaustive in Florians thesis than in the current reference paper. We should add it as an option here for future reader, Any objections?

fsimonis commented on 03-faq.md:
```diff
@@ -39,12 +43,43 @@ through TCP/IP sockets or through MPI ports.
+
+## Can preCICE be used with adaptive meshes?
+
+Or: " Is it correct that preCICE cannot support couplings when the interface mesh is changing its connectivity during the simulation?"
```
```suggestion
Or: "Is it correct that preCICE cannot support couplings when the interface mesh is changing its connectivity during the simulation?"
```

fsimonis commented on 03-faq.md:
```diff
@@ -39,12 +43,43 @@ through TCP/IP sockets or through MPI ports.
+
+Or: " Is it correct that preCICE cannot support couplings when the interface mesh is changing its connectivity during the simulation?"
+
+The mapping setup only happens once, in the initialization. So, currently, in order to modify the mapping setup, one needs to restart the simulation, which would be inefficient. However, a dissertation in progress studies this issue and preCICE will have such a feature in the future.
```
```suggestion
Currently, the mapping setup only occurs once during the initialization of preCICE. Thus, in order to modify the mapping one must restart the simulation. It depends on a case-by-case basis whether this costly method is acceptable or not.
A dissertation in progress studies this issue and preCICE will have such a feature in the future.
```

uekerman commented on 03-faq.md:
```diff
@@ -39,12 +43,44 @@ through TCP/IP sockets or through MPI ports.
+
+## Can preCICE be used for one-way coupling?
+
+Yes! Several people are doing this already and we also kind of do this in our [OpenFOAM-CalculiX FSI tutorial](https://github.com/precice/precice/wiki/Tutorial-for-FSI-with-OpenFOAM-and-CalculiX), where we have two one-way couplings (as we use different meshes). We also often use this for debugging. Nothing changes apart from the configuration file.
```
```suggestion
Yes! Several people are doing this already.
```

uekerman replied:
Our example, I would not consider a one-way coupling. Might only confuse people.

uekerman commented on 03-faq.md:
```diff
@@ -31,6 +31,10 @@ to the other. The simplest method uses the nearest-neighbor's value, while
 
+## Does partitioned coupling introduce error?
+
+Partitioned coupling can indeed lead to errors such as the Added-Mass Effect. However, preCICE provides implicit coupling algorithms to treat the error introduced.
```
Not all errors coming from the partitioning can be cured by implicit coupling.

uekerman commented on 03-faq.md:
```diff
@@ -39,12 +43,44 @@ through TCP/IP sockets or through MPI ports.
 
+## Can the individual solvers run on two different platforms on two different machines in/out of a network?
+
+Yes, at least as long as communication over TCP/IP is possible. The participants find each other using a network address, which is written in a file and read once in the beginning of the simulation. For this reason, all the participants need to be started from the same directory.
```
No need to start the participants from the same directory

uekerman commented on 03-faq.md:
```diff
@@ -39,12 +43,44 @@ through TCP/IP sockets or through MPI ports.
+
+Yes! Several people are doing this already and we also kind of do this in our [OpenFOAM-CalculiX FSI tutorial](https://github.com/precice/precice/wiki/Tutorial-for-FSI-with-OpenFOAM-and-CalculiX), where we have two one-way couplings (as we use different meshes). We also often use this for debugging. Nothing changes apart from the configuration file.
+
+Even though there may be simpler ways to do one-way coupling, it may still be beneficial to use preCICE here: flexible data mapping methods and parallel communication (your coupling data might be too large to write to disk).
```
```suggestion
Even though there may be other ways to do one-way coupling, it may still be beneficial to use preCICE here: flexible data mapping methods, parallel communication (your coupling data might be too large to write to disk), and already adapted solvers.
```

uekerman commented on 03-faq.md:
```diff
@@ -39,12 +43,44 @@ through TCP/IP sockets or through MPI ports.
+Or: "Is it correct that preCICE cannot support couplings when the interface mesh is changing its connectivity during the simulation?"
+
+Currently, the mapping setup only occurs once during the initialization of preCICE. Thus, in order to modify the mapping one must restart the simulation. It depends on a case-by-case basis whether this costly method is acceptable or not.
+A dissertation in progress studies this issue and preCICE will have such a feature in the future.
```
```suggestion
A dissertation in progress studies dynamically changing coupling meshes, such that preCICE will be able to handle adaptive meshes in the future.
```

uekerman replied:
It's a bit unclear to what you refer with "this issue" and "such a feature".

davidscn committed:
Create configuration-mapping.md

The RBF mapping description is much more exhaustive in Florian's thesis than in the current reference paper. We should add it as an option here for future reader, Any objections?

uekerman commented on pages/docs/configuration/configuration-mapping.md:
```diff
@@ -64,7 +64,7 @@ Concerning mapping methods, preCICE offers three variants:
 ## Radial-basis function mapping
 
-Radial basis function mapping computes a global interpolant on one mesh, which is then evaluated at the other mesh. The global interpolant is formed by a linear combination of radially-symmetric basis functions centered on each vertex, enriched by one global linear polynomial. For more information, please refer, e.g., to [this paper](https://www.researchgate.net/publication/317902743_Radial_Basis ...
+Radial basis function mapping computes a global interpolant on one mesh, which is then evaluated at the other mesh. The global interpolant is formed by a linear combination of radially-symmetric basis functions centered on each vertex, enriched by one global linear polynomial. For more information, please refer, e.g., [Florian's thesis](https://elib.uni-stuttgart.de/bitstream/11682/10598/3/Lindne ...
```
```suggestion
Radial basis function mapping computes a global interpolant on one mesh, which is then evaluated at the other mesh. The global interpolant is formed by a linear combination of radially-symmetric basis functions centered on each vertex, enriched by one global linear polynomial. For more information, please refer, e.g., to [Florian's thesis](https://elib.uni-stuttgart.de/bitstream/11682/10598/3/Lindner%20-%20Data%20Transfer%20in%20Partitioned%20Multi-Physics%20Simulations.pdf) (pages 37 ff.) or to [this paper](https://www.researchgate.net/publication/317902743_Radial_Basis_Function_Interpolation_for_Black-Box_Multi-Physics_Simulations) and the reference therein.
```

davidscn committed:
Update pages/docs/configuration/configuration-mapping.md

Co-authored-by: Benjamin Uekermann <benjamin.uekermann@gmail.com>
